lib/util.js

```diff
@@ -758,46 +758,46 @@ exports.p = exports.deprecate(function() {
   for (var i = 0, len = arguments.length; i < len; ++i) {
     console.error(exports.inspect(arguments[i]));
   }
-}, 'util.p: Use console.error() instead');
+}, 'util.p is deprecated. Use console.error instead.');
 
 
 exports.exec = exports.deprecate(function() {
   return require('child_process').exec.apply(this, arguments);
-}, 'util.exec is now called `child_process.exec`.');
+}, 'util.exec is deprecated. Use child_process.exec instead.');
 
 
 exports.print = exports.deprecate(function() {
   for (var i = 0, len = arguments.length; i < len; ++i) {
     process.stdout.write(String(arguments[i]));
   }
-}, 'util.print: Use console.log instead');
+}, 'util.print is deprecated. Use console.log instead.');
 
 
 exports.puts = exports.deprecate(function() {
   for (var i = 0, len = arguments.length; i < len; ++i) {
     process.stdout.write(arguments[i] + '\n');
   }
-}, 'util.puts: Use console.log instead');
+}, 'util.puts is deprecated. Use console.log instead.');
 
 
 exports.debug = exports.deprecate(function(x) {
   process.stderr.write('DEBUG: ' + x + '\n');
-}, 'util.debug: Use console.error instead');
+}, 'util.debug is deprecated. Use console.error instead.');
 
 
 exports.error = exports.deprecate(function(x) {
   for (var i = 0, len = arguments.length; i < len; ++i) {
     process.stderr.write(arguments[i] + '\n');
   }
-}, 'util.error: Use console.error instead');
+}, 'util.error is deprecated. Use console.error instead.');
 
 
-exports.pump = exports.deprecate(function(readStream, writeStream, callback) {
+exports.pump = exports.deprecate(function(readStream, writeStream, cb) {
```

Comment:
@silverwind Ah, that was added to fix the line length issue, when I was calling `internalUtil`'s `deprecate`. I ll fix it.